app/src/main/java/com/anysoftkeyboard/keyboards/Keyboard.java

```diff
@@ -458,6 +458,13 @@ public Key(
             centerY = y + height / 2;
         }
 
+        public int getMultiTapCode(int tapCount) {
+            final int codesCount = getCodesCount();
+            if (codesCount == 0) return KeyCodes.SPACE; // space is good for nothing
```

Comment:
`0` could be an option here? Or maybe it doesn't matter.

Reply:
Probably. I didn't really write that code.

I just needed to implement it at `Keyboard.java`, so I moved it from `app/src/main/java/com/anysoftkeyboard/keyboards/views/PointerTracker.java`, made it public and non-static. It seemed a better solution than copying it.